make npm rebuild works on root folder to ease native rebuild
drop preinstall since it want to fetch node headers
see pre-build target which replaces postinstall.js

Index: package.json
--- package.json.orig
+++ package.json
@@ -18,9 +18,7 @@
     "test-build-scripts": "cd build && npm run test",
     "test-agent-host-e2e-coverage": "node scripts/agent-host-e2e-coverage.ts",
     "check-cyclic-dependencies": "node build/lib/checkCyclicDependencies.ts out",
-    "preinstall": "node build/npm/preinstall.ts",
-    "postinstall": "node build/npm/postinstall.ts",
-    "compile": "npm-run-all2 -lp compile-client compile-copilot",
+    "compile": "npm run compile-client",
     "compile-client": "npm run gulp compile",
     "compile-copilot": "npm --prefix extensions/copilot run compile",
     "build-fast": "npm-run-all2 -lp transpile-client build-fast-extensions compile-copilot",
